### ISSUE TITLE
[Feature Request] Add English (United Kingdom) language

### ISSUE THREAD

**Issue by PopuriAO29** [ip7z/7zip#18]
![7Zip-EnGBFlag](https://github.com/ip7z/7zip/assets/36154203/03a562c7-bcb5-492e-84f4-69a025410f69)

## Is your feature request related to a problem? Please describe.

7-Zip does have English language for default. However, for some reasons, most of the words are using American text because English (United Kingdom) language is not present.

## What you want

I would like to add English (United Kingdom) Language for 7-Zip. Because the flag is literally United Kingdom, and most words are different, such as Favoruites. as the American text is Favorites (in case on the menu bar).

## Describe alternatives you've considered

Possibly, English (English) is the default language. The reason there are various region language for English regions such as United Kingdom.

## Additional Information

Per following this: https://sourceforge.net/p/sevenzip/feature-requests/1600/

I also made English UK Flag with Logo, the logo is credited to the Nurudin.

## Additional Context

Igor said he doesn't want add anymore languages, He asked about British English localization for Windows. To be honest, British English was introduced in Windows 8 for the main language since in 2012. that being said, the Language code is called en-GB.

I Initially make base for **en.ttt** file, then I save the text file for en-GB language. As soon I always update some strings, I always keep updated to make up to date in the future.

Comparing from ImgBurn, I initially made change several strings to matching from ImgBurn and some resembles from Windows 8. through 11. for example, "Computer" text would be "This PC", this will be resembled from Windows 8.1 through 11.

You can clearly see on the changes I made on the above, those are updated to keep my language file for British English.

In other words, some language flag are also appeared in various games such as FIFA 14 and FIFA 23, and other European games that including Language selection, these are flag named it: English (United Kingdom), or British English instead.

Here's what the language flag would looks like in FIFA 23:
![FIFA23-EnglishUKFlag](https://github.com/ip7z/7zip/assets/36154203/ca1d56f2-7c56-4d7a-80b9-0c5da170e6cf)

If you see these, this means it's British English flag, so let me know if you interested this.

## Have you checked that no similar feature request(s) exist?
* [x] I have searched and didn't find any similar feature request.

**Comment by michelleoku** [comment]
(++) **Strongest support** per above.

**Comment by PopuriAO29** [comment]
I am also **strongly support** this for language flag (British English), here's the screenshot from FIFA 09 during language selection.

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/0bc674c0-6ac6-4cf9-986c-07e2f1587bed" />

Over the past months, I've also updated new version for en-GB.txt file, This goes straight on sourceforge that i made feature request [here](https://sourceforge.net/p/sevenzip/feature-requests/1600/) (similar above at Additional Information section)

It would be nice if there is milestone version for 7-zip that wasn't released, such as 25.02 or 26.00, An example shown is about libSDL repository to see milestone [here](https://github.com/libsdl-org/SDL/milestones) (because SDL uses different version, such as 2.20.8. In here for 7-zip, it uses version year such as 25.xx, 24.xx, 23.xx, etc.)
